app/controllers/results_controller.rb

```diff
@@ -4,6 +4,7 @@ class ResultsController < ApplicationController
 
   def index
     @results_view = ResultsView.new(query_parameters: request.query_parameters)
+
     @courses = Course
```

Comment:
This has all been moved into `ResultsView` now. The course bit in #109 and the subjects in #115 which is still in review.